daemon/mgr/cri.go

```diff
@@ -60,15 +63,21 @@ type CriMgr interface {
 type CriManager struct {
 	ContainerMgr ContainerMgr
 	ImageMgr     ImageMgr
+	NetworkMgr   CniMgr
```

Comment:
Could you change the field `NetworkMgr` to be `CNIMgr`?
Since I think NetworkMgr has appeared in daemon side?